bee-tangle/src/solid_entry_point.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! The solid_entry_point module defined the [SolidEntryPoint] type which represents
+//! an already solidified message in the tangle.
```

Comment:
I feel like this definition is a bit off. The other one you use later in more correct.

Reply:
What do you think would be a better definition?

Reply:
You wrote an actual good definition for the type just below.